### PR TITLE
Please comment

### PR DESCRIPTION
**Pull request by warvariuc**
I did some changes and i need your feedback on them.

```
Now not accepting unescaped new line characters in a content line.
When folding a line - preserve space at the end, if present.
Modified the tests accordingly.
```

The changes are arguable, so what do you think?

### CODE REVIEW

File: src/icalendar/parser.py [hunk 1]
```diff
@@ -291,6 +258,8 @@ class Contentline(unicode):
     """
     def __new__(cls, value, strict=False, encoding=DEFAULT_ENCODING):
         value = to_unicode(value, encoding=encoding)
+        assert u'\n' not in value, ('Content line can not contain unescaped '
```
Comment (thet) [inline src/icalendar/parser.py:261]: wouldn't that match also `u'\\n'`?

Comment (warvariuc) [inline src/icalendar/parser.py:261]:
```
>>> u'\n' in u'first line\nsecond line'
True
>>> u'\n' in u'first line\\nsecond line'
False
>>> 
```

Comment (thet) [inline src/icalendar/parser.py:261]: tnx

File: src/icalendar/parser.py [hunk 2]
```diff
@@ -64,50 +63,20 @@ def foldline(text, length=75, newline='\r\n'):
...
-#                replace_whitespace=False
-#                )
-#            )
+    assert isinstance(line, unicode)
+    assert u'\n' not in line
+
+    ret_line = u''
+    byte_count = 0
+    for char in line:
+        char_byte_len = len(char.encode('utf-8'))
```
Comment (rnixx) [inline src/icalendar/parser.py:72]: please use DEFAULT_ENCODING instead of 'utf-8'

**Pull request merged**